site/docs/reference/Connectors/materialization-connectors/Snowflake.md

```diff
@@ -107,4 +107,10 @@ For example, if you materialize a collection with a key of `/user_id`, it will t
 This is because most materializations tend to be roughly chronological over time, and that means that data is written to Snowflake in roughly `/date` order.
 
 This means that updates of keys `/date, /user_id` will need to physically read far fewer rows as compared to a key like `/user_id`,
-because those rows will tend to live in the same micro-partitions, and Snowflake is able to cheaply prune micro-partitions that aren't relevant to the transaction.
\ No newline at end of file
+because those rows will tend to live in the same micro-partitions, and Snowflake is able to cheaply prune micro-partitions that aren't relevant to the transaction.
+
+## Notes
```

Comment:
Same as in Postgres docs, I think this heading can come out and "reserved word" can be the h2